server/models/post.model.js

```diff
@@ -0,0 +1,43 @@
+const { Sequelize, DataTypes } = require('sequelize')
+
+module.exports = (sequelize, Sequelize) => {
+    const Post = sequelize.define(
+        'posts', {
+            id: {
+                type: DataTypes.INTEGER,
+                allowNull: true,

```

Comment:
Why did you allow null for the id? It should not be nullable, take a look the other columns as well

Reply:
If we set allowNull as false, then does id increase from 1 automatically?